dipy/workflows/base.py

```diff
@@ -0,0 +1,124 @@
+from dipy.fixes import argparse as arg
+from dipy.utils.optpkg import optional_package
+
+import inspect
+
+ndoc, has_ndoc, _ = optional_package('numpydoc')
```

Comment:
@matthieudumont it seems we cannot avoid removing the numpydoc dependency. We tried to install dipy_recognize_bundles in another computer (osx yosemite) and it required both numpydoc and sphinx to be installed.
Can you copy only the parts that actually parse the docstring (especially Parameters section) and put them all in a file inside dipy/workflows. Let me know what you think.